static/style.css

```
@@ -1,7 +1,239 @@
+    --div-gradient: rgba(255, 255, 255, 0.8), rgba(241, 241, 241, 0.6) 49%, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0.6) 49%;
+    --div-border: rgba(255, 255, 255, 0.685);
+    --div-scroll-thumb:rgba(255, 192, 203, 0.562);
+    --h2-shadow: rgba(255, 192, 203, 0.349);
+    --h2-gradient: transparent, rgba(255, 255, 255, 0.425);
+
+    --background-anim-center: rgb(0, 225, 255);
+    --background-anim-shadows: -250px -250px hotpink, -250px 0px red, -250px 250px orange, 0px -250px yellow, 0px 250px lime, 250px -250px turquoise, 250px 0px indigo, 250px 250px violet;
 }
+body{
+    background: var(--body-bacground-color);
+    overflow-x: hidden;
+}
+h1{
+    font-weight: bold;
+    text-align: center;
+    font-size: 50;
+    background: var(--title-gradient);
+    color: var(---title-text-color);
+    margin-left: 35%;
+    margin-right: 35%;
+    padding: 20px;
+    -moz-padding:2px;
+    border-radius: 35px 0 35px 0;
+    -webkit-box-shadow: 5px 5px 1px var(--title-shadow);
+            box-shadow: 5px 5px 1px var(--title-shadow);
+}
+
+p{
+    color:var(--text-color);
+    position:relative;
+    background: var(--p-gradient);
+    margin-left: 75px;
+    margin-right: -40px;
+    padding: 20px 40px 20px 10px;
+    border-bottom: var(--p-border) solid;
+    border-width: 0px 2px 2px 0px;
+}
+p::before{
+    content: '';
+    background: var(--p-tag);
+    position:absolute;
+    left: -20px;
+    top: 0px;
+    display: block;
+    width: 20px;
+    height: 100%;
+    border-bottom: 2px var(--p-border) solid;
+}
+small{
+    color: var(--text-color);
+    margin-left: 100px;
+    margin-right: -40px;
+    background: var(--p-gradient);
+    padding: 10px;
+    display: block;
+    border-bottom: var(--p-border) solid;
+}
+h2{
+    color: var(--text-color);
+    text-align: center;
+    background-image: -webkit-linear-gradient( var(--h2-gradient));
+    background-image: -o-linear-gradient(var(--h2-gradient));
+    background-image: linear-gradient(var(--h2-gradient));
+    padding-top: 1em;
+    margin-left: 40%;
+    margin-right: 40%;
+    height: 2em;
+    -webkit-box-shadow: 5px 5px 15px var(--h2-shadow);
+            box-shadow: 5px 5px 15px var(--h2-shadow);
+}
+div{
+    background-image: -o-repeating-linear-gradient(135deg, var(--div-gradient)) ;
+    background-image: repeating-linear-gradient(-45deg, var(--div-gradient)) ;
+    line-height: 40px;
+    
+    margin-left: 10%;
+    margin-right: 10%;
+    text-align:center;
+    padding: 10px;
+    border: var(--div-border) 2px solid;
+    border-radius: 0 50px 0 50px;
+    /*max-height: 1000px;
+    overflow-y: scroll;
+    row-gap: 20px;
+    scrollbar-width: thin;
+    scrollbar-color: var(--div-scroll-thumb);*/
+}
+/*div::-webkit-scrollbar{
+    width: 1em;
+}
+div::-webkit-scrollbar-track{
+    margin-block: 30px;
+    right: -10px;
+    margin-right: 10px;
+    
+}
+div::-webkit-scrollbar-thumb{
+    background-color: var(--div-scroll-thumb);
+    border-radius: 1em;
+}*/
```

I just personally don't like commented out code, I think if we really need to get an old version back we can use git's history for that ^^

```suggestion
}
```